uk.et1.big-finish-downloader.yaml

```diff
@@ -23,7 +23,8 @@ modules:
       - chmod 755 /app/bin/big-finish-downloader-gtk
       - glib-compile-schemas /app/share/glib-2.0/schemas
       - mkdir -p /app/share/app-info/xmls/
-      - gzip -c share/metainfo/uk.et1.big-finish-downloader.metainfo.xml > share/app-info/xmls/uk.et1.big-finish-downloader.xml.gz
+      - gzip share/metainfo/uk.et1.big-finish-downloader.metainfo.xml
+      - mv share/metainfo/uk.et1.big-finish-downloader.metainfo.xml.gz  /app/share/app-info/xmls/uk.et1.big-finish-downloader.xml.gz
```

Comment:
thats not what you want to do.

what you want to do is install `share/metainfo/uk.et1.big-finish-downloader.metainfo.xml` into `${FLATPAK_DEST}/share/metainfo`.